modules/fscloud/outputs.tf

```diff
@@ -0,0 +1,39 @@
+##############################################################################
+# Outputs
+##############################################################################
+
+output "cis_instance_name" {
+  description = "CIS instance name"
+  value       = module.cis_instance.cis_instance_name
+}
+
+output "cis_instance_guid" {
+  description = "GUID of CIS instance"
+  value       = module.cis_instance.cis_instance_guid
+}
+
+output "cis_instance_id" {
+  description = "CRN of CIS instance"
+  value       = module.cis_instance.cis_instance_id
+}
+
+output "cis_instance_status" {
+  description = "Status of CIS instance"
+  value       = module.cis_instance.cis_instance_status
+}
+
+output "cis_domain" {
+  description = "CIS Domain details"
+  value       = module.cis_instance.cis_domain
+}
+
+output "cis_glb_id" {
+  description = "ID of CIS GLB"
```

Comment:
This will look better- `GLB ID of CIS`

Reply:
In GLB module, other outputs were mentioned like:
ID of CIS helath checks / ID of CIS origin pools etc. 

So kept it in a same way.